lib/cpp/antaris_api_client.cc

```diff
@@ -463,6 +465,21 @@ void *start_callback_server(void *thread_param)
     // clients. In this case it corresponds to an *synchronous* service.
     builder.RegisterService(&ctx->callback_service_handle);
 
+    if (g_KEEPALIVE_ENABLE == ENABLED) {
+        // Sample way of setting keepalive arguments on the server. Here, we are
+        // configuring the server to send keepalive pings at a period of 10 minutes
+        // with a timeout of 20 seconds. Additionally, pings will be sent even if
+        // there are no calls in flight on an active HTTP2 connection. When receiving
+        // pings, the server will permit pings at an interval of 10 seconds.
+        builder.AddChannelArgument(GRPC_ARG_KEEPALIVE_TIME_MS,
+                                   10 * 60 * 1000 /*10 min*/);
```

Comment:
Can all these literals be defined as constants (#define...) at one place ? Easy for future changes.